modules/cluster-logging-deploy-cli.adoc

```diff
@@ -373,31 +282,68 @@ elasticsearch-cdm-x6kdekli-3   1/1     1            0           6m44s
 The number of primary shards for the index templates is equal to the number of Elasticsearch data nodes.
 ====
 
-.. Create the instance:
+. Apply the `ClusterLogging` custom resource (CR) by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc apply -f <filename>
 ----
 +
-For example:
+This creates the {logging} components, the `Elasticsearch` CR and components, and the Kibana interface.
+
+.Verification
+
+. Verify the {es-op} installation:
 +
 [source,terminal]
 ----
-$ oc create -f olo-instance.yaml
+$ oc get csv --all-namespaces
 ----
 +
-This creates the {logging} components, the `Elasticsearch` custom resource and components, and the Kibana interface.
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                            DISPLAY                  VERSION               REPLACES   PHASE
```

Comment:
```suggestion
NAMESPACE                                               NAME                                            DISPLAY                  	VERSION               		REPLACES   	   PHASE
```
These titles aren't aligned once you get to Version if you scroll over. Looks like another tab or two is needed.